### ISSUE TITLE
Enhanced feedback after computation wanted

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-robotlog2db#37]
This is the current feedback after computation:

"`All test results are written to database successfully.`"

It would be more user friendly to have this final feedback available in more detail.

New counters should be introduced, counting:
* all tests found in XML result file,
* test successfully written to database,
* tests with any errors while trying to write (e.g. invalid or missing values).

Possible feedbacks now can be:

* "`All <number> test results are written to database successfully.`"
* "`All <number> test results are appended to database successfully.`"
* "`<number_success> of <number_all> test results are written to database successfully. <number_failed> test results are skipped because of errors.`"
* "`<number_success> of <number_all> test results are appended to database successfully. <number_failed> test results are skipped because of errors.`"
* Final ststistics:
  e.g.:
```
  component A : 5 test results
  component B : 6 test results
  component C : 7 test results
```
This would be very helpful, because now the database application can also be tested without the database being involved. We would have more informations available: Because the expectation would be that the database application counts the same number of test results than the web application. In case of deviating values we immediately detect that there must be an error.

This is related to:
https://github.com/test-fullautomation/robotframework-robotlog2db/issues/26

Same requirement for PyTestLog2DB.

**Comment by HolQue** [comment]
In case of values are missing (e.g. versions), the app defines default values. But that this happens is not mentioned in console.

A feedback is e.g.:

Created test execution result for variant '...' - version '20230228_155346' successfully:

This sentence is OK and should stay, but in case of any default values are involved I would prefer to have for every default value an additional line like this on top:

Setting version to default value '20230228_155346'.

Then it would be more obvious that it has not been the user who has set the value.

**Comment by test-fullautomation** [comment]
solved with #45
